liblouis/Makefile.am

```diff
@@ -22,6 +22,10 @@ liblouis_la_LDFLAGS =	\
 	-version-info $(LIBLOUIS_CURRENT):$(LIBLOUIS_REVISION):$(LIBLOUIS_AGE) -no-undefined \
 	$(LTLIBINTL)
 
+if OSX
+liblouis_la_LDFLAGS += -Wl,-install_name,@rpath/liblouis.dylib
```

Comment:
Why is this needed? Doesn't libtool handle this for you?

Reply:
This was needed when I still had the separate "table-dev" library. Libtool doesn't do this for you.

This line can be removed now that all the code is inside the main library. I just forgot to remove it.